Oracle/README.md

```diff
@@ -31,11 +31,11 @@ The following are the steps to execute the DDL Code Generation. They can be exec
 
 ## **For Linux/Unix:**
 
-There are two options available to execute the extraction script based on your environment and the tools you have installed: Sql*Plus and sqlcl.
+There are two options available to execute the extraction script based on your environment and the tools you have installed: SQL\*Plus and sqlcl.
 
-Sql*Plus: If you have Oracle installed on your machine, you can use Sql*Plus, which is a command-line tool for Oracle Database. It provides a powerful and interactive interface to execute SQL statements and scripts. 
+SQL\*Plus: If you have Oracle installed on your machine, you can use SQL\*Plus, which is a command-line tool for Oracle Database. It provides a powerful and interactive interface to execute SQL statements and scripts. 
 
-sqlcl: If you don't have Oracle installed locally or prefer a modern command-line interface, you can use sqlcl (SQL Developer Command Line). Sqlcl is a free, lightweight tool provided by Oracle that offers a feature-rich SQL and PL/SQL scripting environment. It provides similar functionality to Sql*Plus but with additional enhancements. 
+sqlcl: If you don't have Oracle installed locally or prefer a modern command-line interface, you can use sqlcl (SQL Developer Command Line). Sqlcl is a free, lightweight tool provided by Oracle that offers a feature-rich SQL and PL/SQL scripting environment. It provides similar functionality to SQL\*Plus but with additional enhancements. 

```

Comment:
Can you add links to both of these tools, specially so clients can check any requirements needed